db/data/20231213121605_cleanup_bd_org_data.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CleanupBdOrgData < ActiveRecord::Migration[6.1]
```

Comment:
Can name this specific to ICDDRB org

Reply:
Was not sure how much to reveal since this is a public repo. Will change.